src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/tableResources/table.jelly

```diff
@@ -55,22 +55,7 @@ THE SOFTWARE.
         "stateSave": true
       }'
     >
-      <colgroup>
-        <!-- index -->
-        <col class="col-width-0 text-end" />
-        <!-- resouce -->
-        <col class="col-width-5 text-end" />
-        <!-- status -->
-        <col class="col-width-2 text-end" />
-        <!-- timestamp -->
-        <col class="col-width-1 text-end" />
-        <!-- labels -->
-        <col class="col-width-5 text-end" />
-        <!-- properties -->
-        <col class="col-width-5 text-end" />
-        <!-- actions -->
-        <col action="col-width-1 text-end" />
-      </colgroup>
```

Comment:
I haven't done anything with the Jenkins CSS system, yet. Why can this section be removed? Because setting the width is done by the `width-100` class?
If so, what about the `text-end` property?

Reply:
I like more to set the property per column separatelly, because so it is better readeable.

text-end - Good question. I will check it.